bin/winsvc_wrapper..ml

```diff
@@ -0,0 +1,7 @@
+let formatter = Format.err_formatter
+
+let run _name main = main ()
+
+let install _name _display _text _arguments = Logs.err (fun f -> f "Cannot install a service on non-Windows systems.")
```

Comment:
```suggestion
let install _name _display _text _arguments = failwith "Cannot install a service on non-Windows systems."
```
We shouldn't return success from this function if it can't install.